s3/src/main/scala/org/apache/pekko/stream/connectors/s3/impl/S3Stream.scala

```diff
@@ -1177,11 +1177,15 @@ import scala.util.{ Failure, Success, Try }
 
         import conf.multipartUploadSettings.retrySettings._
 
-        SplitAfterSize(chunkSize, chunkBufferSize)(atLeastOneByteString)
-          .via(getChunkBuffer(chunkSize, chunkBufferSize, maxRetries)) // creates the chunks
-          .mergeSubstreamsWithParallelism(parallelism)
+        val source1: SubFlow[Chunk, NotUsed, Flow[ByteString, ByteString, NotUsed]#Repr, Sink[ByteString, NotUsed]] =
```

Comment:
I presume that this type annotation was auto generated from Intellij and if so it likely can be simplified a bit more.